tests-hivemq-operator/src/integrationTest/resources/values/ese-extension-values.yaml

```diff
@@ -1,7 +1,3 @@
-global:
```

Comment:
This is just a cleanup or?

Reply:
The commit says `Remove unnecessary pspEnabled value for legacy operator tests`. So yes, just a cleanup of test resources.